internal/auth0/organization/resource_connection_test.go

```diff
@@ -199,3 +265,70 @@ func TestAccOrganizationConnection(t *testing.T) {
 		},
 	})
 }
+
+func TestAccOrganizationConnectionSAB(t *testing.T) {
```

Comment:
Could these just be different steps inside the main `TestAccOrganizationConnection` ?

Reply:
I was thinking I would put them here, but keeping them separated out made running the tests MUCH faster. Now that they are working, I can move them back